titus_isolate/isolate/cpu.py

```diff
@@ -33,7 +147,7 @@ def assign_threads(cpu, workload):
     return claimed_threads + assign_threads(cpu, Workload(workload.get_id(), thread_count, workload.get_type()))
 
 
-def free_threads(cpu, workload_id):
+def free_threads_greedy(cpu, workload_id):
```

Comment:
Dead code now.  Let's just remove the method.

Reply:
As discussed in person, since we want the fallback I kept it.